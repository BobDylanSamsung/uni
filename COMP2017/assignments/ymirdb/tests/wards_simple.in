set a 1 2 3
set b 4 5 6
set c 9 8 7 0
forward a
backward a
forward b
backward b
forward c
backward c
bye
