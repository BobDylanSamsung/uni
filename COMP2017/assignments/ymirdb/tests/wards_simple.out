ok

ok

ok

nil

nil

nil

nil

nil

nil

bye
